tests/python/contrib/test_arm_compute_lib/test_conv2d.py

```diff
@@ -21,15 +21,15 @@
 import tvm
 from tvm import relay
 
-from .infrastructure import (
+from test_arm_compute_lib.infrastructure import (
```

Comment:
It's not obvious to me why this change is related.

Reply:
to be able run tests as python -m test_arm_compute_lib.test_*

Reply:
I think this should be in a separate patch then - doesn't relate to the change to annotate target.